src/features/goblins/tailor/actions/items.ts

```diff
@@ -0,0 +1,237 @@
+import { CONFIG } from "lib/config";
+import { ERRORS } from "lib/errors";
+import { BumpkinItem, BumpkinPart } from "features/game/types/bumpkin";
+import { Release } from "../TabContent";
+import { MOCK_SHOP } from "./mockShopItems";
+import { metamask } from "lib/blockchain/metamask";
+
+const API_URL = CONFIG.API_URL;
+
+export interface BumpkinShopItem {
+  name: BumpkinItem;
+  tokenId: number;
+  totalMinted?: number;
+  part?: BumpkinPart;
+  price?: number;
+  releases: Release[];
+  currentRelease?: Release;
+}
+
+export type Shop = {
+  items: BumpkinShopItem[];
+};
+
+const CACHE_KEY = "bumpkin.shopItems";
+const CACHE_MINUTES = 5;
+
+type Cache = {
+  cachedAt: number;
+  items: Item[];
+};
+
+export type Item = {
+  id: number;
+  releases: Release[];
+  name: BumpkinItem;
+};
+
+export function isCommon(releases: Release[]) {
```

Comment:
can you explain the naming for this function? isCommon refers to the rarity of the item? i'm a little confused 😳

Reply:
`isCommon` is in reference to items that will always be available to purchase on our Bumpkins.io website. These will be basic items like some hair styles, some shirts etc. they will have an unlimited supply. Once they're released they will always be available to mint.

Reply:
Thanks @spencerdezartsmith, now i understand it.🚀